### PR TITLE
HELM-3: Use Helm dependency management

### PR DESCRIPTION
**Pull request by ASHISH932**
* Added mysql as a dependency of chart

### CODE REVIEW

**Comment by slayerjain** [comment]
You haven't updated the docs yet ;)

**Comment by nehagup** [comment]
- The new Chart format expresses dependencies using the charts/ directory.

- I think there should be descriptions of options in values.yaml.

File: requirements.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+dependencies:
+  - name: mysql
+    repository: https://kubernetes-charts.storage.googleapis.com
+    version: 1.2.0
+    import-values:
```
Comment (nehagup) [inline requirements.yaml]: Why is this needed?

File: requirements.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+      - child: default.data
```
Comment (nehagup) [inline requirements.yaml]: I don't see child chart or values.

File: requirements.yaml (hunk 1 above)
Comment (ASHISH932) [reply requirements.yaml]: This is the way to tell helm that this chart depends on mysql, and to deploy it before the chart

File: requirements.yaml (hunk 2 above)
Comment (ASHISH932) [reply requirements.yaml]: I would remove this line, it is of no use, for now, would add it later if needed.

**Comment by ASHISH932** [comment]
> * I think there should be descriptions of options in values.yaml.

By options what do you mean?

File: requirements.yaml (hunk 1 above)
Comment (nehagup) [reply requirements.yaml]: Adding it as a dependency will automatically deploy it before the chart. Import-Values doesn't help that.

File: requirements.yaml (hunk 2 above)
Comment (nehagup) [reply requirements.yaml]: Sure, also the parent is also not required for now I think.

**Comment by nehagup** [comment]
> > * I think there should be descriptions of options in values.yaml.
> 
> By options what do you mean?

By options I mean the default values and the other options should be well-described and documented in values.yaml file, you can take some reference from helm/stable charts.

File: requirements.yaml (hunk 2 above)
Comment (nehagup) [reply requirements.yaml]: I think you removed that in the last commit. So, fine for now.

File: requirements.yaml (hunk 1 above)
Comment (ASHISH932) [reply requirements.yaml]: Removed that in the last-commit
